Docs terminology cleaning (#78)
onflow/flow: docs/content/token/staking2/stakers.mdx

|------------|--------|---------|
| **amount** | UFix64 | The quantity of rewarded FLOW tokens to add to the user's stake. |

> Since rewarded tokens are considered to be unstaked for the purposes of the vesting schedule,

Reviewer: ```suggestion
Since rewarded tokens are considered to be unlocked for the purposes of the vesting schedule,
```

and staked tokens are considered locked, staking rewarded tokens marks the staked tokens as locked

> and marks an equal amount of tokens in the user's locked account as unstaked.

Reviewer: ```suggestion
and marks an equal amount of tokens in the user's locked account as unlocked.
```

|------------|--------|---------|
| **amount** | UFix64 | The quantity of rewarded FLOW tokens to withdraw from the proxy. |

> Since rewards are considered to be unstaked. This marks an equal amount in your locked account as unstaked.

Reviewer: ```suggestion
Since rewards are considered to be unlocked for the purposes of vesting. This marks an equal amount in your locked account as unlocked.
```

> You can immediately withdraw your unstaked reward tokens from your account after withdrawing rewards.

Reviewer: ```suggestion
You can immediately withdraw your unlocked reward tokens from your account after withdrawing rewards.
```
